src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -130,6 +131,16 @@ export class CreateIndexPatternWizard extends Component {
     });
 
     const createdId = await emptyPattern.create();
+    if (!createdId) {
+      const confirmMessage = i18n.translate('kbn.management.indexPattern.titleExistsLabel', { values: { title: this.title },
+        defaultMessage: 'An index pattern with the title \'{title}\' already exists.' });
+      try {
```

Comment:
Why does this get a try / catch block? Just curious.

Can you describe the case where `createId` isn't returned?

Reply:
https://github.com/elastic/kibana/pull/36348/files/785e371d68f73e844a0a15454e4113d0a4654d0d#diff-c7137de69ed88ebbd5a4702660f709eaL376

create method would before show this modal, if override was not allowed allowed. now it returns false in that case. (so createdId is false when indexpattern already exists and overriding is not allowed)

i just moved that code outside of index patterns service.